src/sql/workbench/services/notebook/common/sqlSessionManager.ts

```diff
@@ -394,29 +393,23 @@ export class SQLFuture extends Disposable implements FutureInternal {
 	}
 
 	private convertToHtmlTable(columns: IDbColumn[], d: QueryExecuteSubsetResult): string {
-		let data: SQLData = {
-			columns: columns.map(c => escape(c.columnName)),
-			rows: d.resultSubset.rows.map(r => r.map(c => c.displayValue))
-		};
-		let table: HTMLTableElement = document.createElement('table');
-		table.createTHead();
-		table.createTBody();
-		let hrow = <HTMLTableRowElement>table.insertRow();
-		// headers
-		for (let column of data.columns) {
-			let cell = hrow.insertCell();
-			cell.innerHTML = column;
+		let htmlString = '<table>';
```

Comment:
something like 'resultSet' instead of 'd' as method parameter?

Reply:
cool, renamed it. thanks!